agent/metadata_service.go

```diff
@@ -103,6 +104,10 @@ Return fake data for programs that depend on data from the metadata service.
 These fields are constructed to be obviously wrong and would never be found in the
 production environment.
 */
+func (mds *metadataService) getServices(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprint(w, "fake-meta-data")
```

Comment:
yeah, I've figured that out. It actually doesn't matter what it returns as long as it's not an error.